json_schemas/framework-agreement-details.json

```diff
@@ -15,6 +15,57 @@
       "minLength": 1,
       "type": "string"
     },
+    "frameworkRefDate": {
```

Comment:
good point - @TheDoubleK - this was just transposed from the script - thoughts on better naming?